trieste/acquisition/function/function.py

```diff
@@ -17,6 +17,7 @@
 """
```

Comment:
if you merge in develop you'll see that we now have a new file for active learning in this same directory, `active_learning.py`